src/Mvc/test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesWithBasePathTest.cs

```diff
@@ -323,7 +323,7 @@ public async Task PagesInAreas_CanGenerateLinksToControllersAndPages()
             var expected =
 @"<a href=""/Accounts/Manage/RenderPartials"">Link inside area</a>
 <a href=""/Products/List/old/20"">Link to external area</a>
-<a href=""/Accounts"">Link to area action</a>
+<a href="""">Link to area action</a>
```

Comment:
I think I left this comment as my only feedback on the aspnet/Mvc PR.... why not change the test to do something valid?

I'm not sure I care a whole ton about this part because I think we will have to restore the 2.1/2.2 behaviour. I think it's fine to do that after this PR.